### ISSUE TITLE
Stopped working on the latest windows 11 version :(

### ISSUE THREAD

**Issue by Sirmyself** [valinet/Win11DisableRoundedCorners#53]
![image](https://github.com/valinet/Win11DisableRoundedCorners/assets/11737885/049a3d42-803e-439c-8efb-d286c1b230b5)

The windows version is 
 - w11 22H2;
 - OS Build 22621.2428;
 - Experience Windows Feature Experience Pack 1000.22674.1000.0

**Comment by bibsp** [comment]
I came here to report it and then find someone else already reported it.
Please fix it... I hate rounded corners on desktop apps.

**Comment by LunarEllipse** [comment]
https://github.com/valinet/Win11DisableRoundedCorners/issues/41

**Comment by ackys** [comment]
please update for the latest windows build.

**Comment by nicholasit** [comment]
![image](https://github.com/valinet/Win11DisableRoundedCorners/assets/20179188/486d19aa-8dc9-484a-b9a7-92184b257a9a)

Working here with latest windows update.

I needed to run this in prompt before running Win11DisableOrRestoreRoundedCorners.exe again:

```
cd /d C:\Windows\System32
del uDWM_win11drc.bak
```

**Comment by ackys** [comment]
It is actually working. Thank you.

**Comment by improwise** [comment]
You can also just go to C:\Windows\System32 and delete/rename the uDWM_win11drc.bak file and then run normally for anyone one familiar with the CLI.

**Comment by Dernailer** [comment]
Doesn't work with surface x pro sq1 , win 11 pro  23H2

**Comment by idklololololololol** [comment]
dude it doesn't work again...
![1831283123](https://github.com/valinet/Win11DisableRoundedCorners/assets/153565769/6a63d43a-830c-49a3-8763-5da9a1cc9778)

**Comment by Sirmyself** [comment]
> dude it doesn't work again...

Dude, there are nicer ways to indicate you're having an issue with a batch file someone else worked on for free...

**Comment by anawe22** [comment]
Yeah! Just delete the backup as indicated in [https://github.com/valinet/Win11DisableRoundedCorners/issues/53#issuecomment-1782795550](https://github.com/valinet/Win11DisableRoundedCorners/issues/53#issuecomment-1782795550)
And it works!!! (24H2 26100.2605)
Love you Dude.
